docs/rsdl/rapid-pro-rsdl-abnf.md

```diff
@@ -26,47 +28,43 @@ This grammar uses ABNF as defined by [RFC5234](https://tools.ietf.org/html/rfc52
 ```ABNF
 model               = 1*modelElement
 
-modelElement        = typeDefinition /
-                      serviceDefinition /
-                      enumDefinition
+modelElement        = typeDefinition
+                    / serviceDefinition
+                    / enumDefinition
 ```
 
 ### Type definition
 
 ```ABNF
-
-
 typeDefinition      = "type" "{" typeMember "}"
```

Comment:
@chrisspre This makes `type` case-insensitive, i.e. `tYpE` is also accepted. 

Is this intentional?